src/api/user/session.md

```diff
@@ -467,15 +467,15 @@ Delete all sessions for the authenticated users.
 ```sh
 curl -X DELETE \
   -H "Authorization: token" \
-  https://alekeagle.me/sessions/all?allButSelf=true
```

Comment:
Sorry to have written such terrible documentation, I kind of got sick of writing documentation that it made me want to scream, but according to the [API implementation](https://github.com/AlekEagle/cumulonimbus-api/blob/master/src/api/user/UserSessionEndpoints.ts#L293) the endpoint is indeed /api/user/sessions/all for this specific endpoint